Jenkinsfile

```diff
@@ -48,6 +48,10 @@ install_tabular_all = """
     python3 -m pip install --upgrade -e tabular/[all,tests]
 """
 
+install_multimodal = """
```

Comment:
This will break pip install unless you ensure multimodal is released in pip. `multimodal` must be added to the following files:

https://github.com/awslabs/autogluon/blob/master/.github/workflows/pythonpublish.yml
https://github.com/awslabs/autogluon/blob/master/.github/workflows/pythonpublish_testpypi.yml
https://github.com/awslabs/autogluon/blob/master/.github/workflows/pypi_release.yml
https://github.com/awslabs/autogluon/blob/master/.github/workflows/platform_tests-command.yml#L215

Reply:
It seems only the last file needs to be changed.

Reply:
Added to the last file.

Reply:
Why does only the last file need to be changed?? Without this our pre-release will not make autogluon.multimodal available, which will cause autogluon.text to crash.

Reply:
Could you open the first 3 files and see where to add `multimodal`??? I don't see any other module names in these files.

Reply:
Ditto for the other 2 files

Reply:
Never mind. I have found where to add it.

Reply:
I feel it's kind of unnecessarily complicated to add a new module. There are too many places to add a new name. Do we have a plan to simply this?

Reply:
All added. Thanks.